packages/cmf/src/middlewares/http/index.md

```diff
@@ -119,3 +119,35 @@ dispatch(fetchDataSets());
 |---|---|---|---|
 | url | string | The HEAD url | true |
 | config | object | The rest of action configuration. | false |
+
+# HTTP Error
+
+Most of the time the error managment is centralized. You don't want to handle
+all http error code for each request.
+
+Sometimes you want to handle one particular case. For this you have onError param.
+
+In this case we want to handle 404. If not 404 stay on global HTTP error
```

Comment:
Reading the code and the test case this statement is wrong shouldn't it be 

"In this case we want to handle the 404, yet the 404 is still dispatched and handled in the global error system.